tests/fuzz/runner.py

```diff
@@ -11,7 +11,7 @@
 
 
 def run_fuzzer(fuzzer: str):
-    logging.info(f"Running fuzzer {fuzzer}...")
+    logging.info("Running fuzzer %s...", fuzzer)
```

Comment:
General comment: to the average developer, it is unclear (at a high level)
- which fuzzers currently exist in ClickHouse,
- how they work each (e.g. do they provide generate random SQL, file data, network input, etc. - based on which library exactly?)
- what steps are needed to run them locally (e.g. which options are needed for CMake, which scripts or tests to call per fuzzer, how do they interact with sanitizers (e.g. below I find *SAN_OPTIONS)
- how are `tests/fuzz/*_fuzzer.options` files related and if/how/when developers should change them
- what fuzzing tests currently run in CI and where to look / how to reproduce when a failure was found.

In other words, we would need a README similar to tests/integration/README.md.

EDIT: I just found https://clickhouse.com/docs/en/development/tests#fuzzing and tests/fuzz/README.md. Anyways, both are incomplete and quite messy. It would be nice to consolidate the information into one comprehensive README.md file (in the repository) and then link from the website to that README (one sentence in the "Fuzzing" section is enough).

Reply:
While at it (but I am not sure if you are the right person here) - similar knowledge is ideally documented for OSS fuzz builds which @nickitat [recently fixed](https://github.com/ClickHouse/ClickHouse/pull/67235)). To someone not familiar with fuzzing, it is not clear
- what OSS fuzz tests,
- where fuzzing results can be found,
- how that is different from what we test in our CI,
- who is when and how notified if something goes wrong.

Reply:
I agree that we need more comprehensive guide for libFuzzers, but let's do it separately - our goal here to fix build and make it runnable in our CI - it will help to keep it alive.